Dockerfile

```diff
@@ -7,9 +7,18 @@ COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
 RUN go build -ldflags="-X 'github.com/kanopy-platform/buildah-plugin/internal/version.version=${VERSION}' -X 'github.com/kanopy-platform/buildah-plugin/internal/version.gitCommit=${GIT_COMMIT}' -X 'github.com/kanopy-platform/buildah-plugin/internal/version.pluginType=${PLUGIN_TYPE}'" -o /go/bin/app ./cmd/
+RUN go install github.com/awslabs/amazon-ecr-credential-helper/ecr-login/cli/docker-credential-ecr-login@latest
```

Comment:
I would like to pin to a version, but currently not do-able:
https://github.com/awslabs/amazon-ecr-credential-helper/issues/487

If we want to keep this dependency up to date as we discussed during team meeting, then maybe this is desired. If we want to pin to a version, I can try something else.